website/src/components/ask-ai/AISearch.jsx

```diff
@@ -78,7 +78,7 @@ const AISearch = forwardRef((props, ref) => {
         registerAISearch(query);
 
         try {
-            const apiUrl = "https://hook.eu1.make.com/zw41brw94vfiwuue91riqmoe2ms2nhf6";
+            const apiUrl = "https://ai.appsmith.com/api/v1/docs/query";
             const response = await fetch(apiUrl, {
```

Comment:
The response generated is different for some queries:
<img width="811" alt="GitBranchResponse" src="https://github.com/appsmithorg/appsmith-docs/assets/99252011/cbe850ed-83b6-4963-b02d-d1abc9d8192a">
<img width="813" alt="ProdResponse" src="https://github.com/appsmithorg/appsmith-docs/assets/99252011/816daf27-dcf9-425b-ab6e-64fa2abcaeb6">

Reply:
Steps 4 and 5 generated by the Ask AI are not correct - 
Ask AI 

![AskAI](https://github.com/appsmithorg/appsmith-docs/assets/99252011/81994ec2-fa1c-4bf5-b94d-e2f52db48837)

![AskAi-code](https://github.com/appsmithorg/appsmith-docs/assets/99252011/436b9beb-fe99-4dc8-8d79-85befaf57ef1)

Prod AI - 
![prodAI](https://github.com/appsmithorg/appsmith-docs/assets/99252011/a0395c8c-b8d9-47ca-ac20-ab400649c8de)


![ProdAI-code](https://github.com/appsmithorg/appsmith-docs/assets/99252011/7b7f43b4-c9ae-4bf3-94ae-991e49499ca3)

Reply:
What was the query for this usecase?

Reply:
I guess it was - how to create custom charts using ECharts?

Reply:
Is the Harry's supermart Chart example part of our documentation?